packages/replay-next/components/inspector/PropertiesRenderer.tsx

```diff
@@ -30,10 +30,12 @@ export default function PropertiesRenderer({
   object,
   path,
   pauseId,
+  hidePrototype = false,
 }: {
   object: ProtocolObject;
   path?: string;
   pauseId: ProtocolPauseId;
+  hidePrototype?: boolean;
```

Comment:
Alpha sort props and type attributes :)

Reply:
Should this be in the ESLint config? I saw that `eslint-plugin-sort-keys-fix` was there in the `package.json` but not used in `.eslintrc.json`

Reply:
Possibly. Can't remember if there was a reason for omitting it (in some cases).

Reply:
We can restrict it to `.tsx` files https://www.npmjs.com/package/eslint-plugin-sort-keys-fix#usage. I can open a PR if it makes sense to add it